examples/enricher/src/main/resources/META-INF/properties.xml

```diff
@@ -37,13 +37,13 @@
   </property>
   <property>
     <name>dt.application.EnricherAppWithJSONFile.operator.Enrich.prop.store.fileName</name>
-    <value>/tmp/circleMapping.txt</value>
+    <value>src/main/resources/circleMapping.txt</value>
```

Comment:
maybe for clarity prefix with './' both of the values

Reply:
Done